13-microbiome.Rmd

```diff
@@ -0,0 +1,41 @@
+
+## Goals of Amplicon analysis 
+
+The technologies that are enabling work in microbiome science are the same that are driving the data revolution in biology. Primarily this work is driven by high-throughput DNA sequencing, which is applied for profiling microbial community composition (e.g., marker gene profiling such as 16S or ITS sequencing), functional potential (such as shotgun metagenomic sequencing), and functional activity (such as metatranscriptome sequencing). Other “omics” technologies are now playing an increasing role in microbiome research, such as mass-spectrometry-based metabolomics, which provides profiles of small molecule metabolites in an environment, and metaproteomics which provides more detailed descriptions of functional activities of microbes (and their hosts, if applicable). As a result, bioinformatics software tools are essential to microbiome research. For many microbiome researchers, bioinformatics is an intimidating and challenging aspect of their projects.
+
+
+## Microbiome Analysis with QIIME 2 
+Greg Caporaso's focus for the past decade has been on developing tools to empower researchers to analyze their microbiome data. This work is centered around the QIIME microbiome bioinformatics platform, and currently QIIME 2. The original QIIME, now referred to as QIIME 1, was published in 2010 [CKS+10] and has been cited tens of thousands of times in the primary literature. QIIME 2, which was published in July of 2019 [BRD+19], succeeded QIIME 1 on 1 January 2018. QIIME 2 is better than QIIME 1 in all ways, and QIIME 1 is no longer actively supported. If you have previously used QIIME 1, you should invest time in learning and switching to QIIME 2. If you’re new to QIIME, start with QIIME 2. (When I refer to QIIME in this book, without specifying whether I’m referring to QIIME 1 or QIIME 2, I’m referring to the platform generally.)
```

Comment:
This first sentence feels weird. I definitely appreciate Greg's experience but the learners probably don't need to be sold on how much experience he has. 

Also not sure if the citations will work here? Here's the description for how to add OTTR sources: https://www.ottrproject.org/more_features.html#citing-sources

Reply:
Totally! I have edited that now! 
I also address the citations. Thanks for the resource! I am not just waiting to see if they look correct in the preview